scipy/special/tests/test_basic.py

```diff
@@ -808,11 +808,11 @@ def test_pbwa(self):
         cephes.pbwa(1,0)
 
     def test_pdtr(self):
-        val = cephes.pdtr(0, 1)
+        val = cephes.pdtr(0.0, 1.0)
```

Comment:
You shouldn't need to change all these to `0.0` etc. as they will be casted.

Reply:
ok, changed back.